prisma/migrations/20250114085630_add_credential_model/migration.sql

```diff
@@ -34,12 +50,15 @@ CREATE TABLE "Image" (
 -- CreateTable
 CREATE TABLE "CandidUserInfo" (
     "id" SERIAL NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
```

Comment:
this is interesting are we repurposing the candid workflow for validations?  why edit this table?